src/awkward/_v2/_connect/cling.py

```diff
@@ -583,9 +604,9 @@ def generate(self, compiler, use_cached=True):
 namespace awkward {{
   class {self.class_type()}: public ArrayView {{
   public:
-    {self.class_type()}(ssize_t start, ssize_t stop, ssize_t which, ssize_t* ptrs)
-      : ArrayView(start, stop, which, ptrs) {{ }}
-    {self.class_type()}() : ArrayView(0, 0, 0, 0) {{ }}
+    {self.class_type()}(ssize_t start, ssize_t stop, ssize_t which, ssize_t* ptrs, PyObject* lookup = nullptr, PyObject* generator = nullptr)
```

Comment:
See comment about Iterators (above).

After this, there are a lot more cases of these for all the concrete classes. It _should_ be safe to just remove the defaults now that everything has been written. If there's a compiler error, then that's great—it would otherwise have been a segfault.

Reply:
done